scalafmt-core/shared/src/main/scala/org/scalafmt/internal/Split.scala

```diff
@@ -38,8 +38,8 @@ case class OptimalToken(token: Token, killOnFail: Boolean = false) {
 case class Split(
     modExt: ModExt,
     cost: Int,
-    tag: SplitTag = SplitTag.Active,
-    activeTag: SplitTag = SplitTag.Active,
+    neededTags: Long = SplitTag.Active.mask,
```

Comment:
Not sure how much better the bit mask is than just `Set[SplitTag]` here and `contains` instead of `(neededTags & splitTag.mask) != 0`

Reply:
done. i thought i had seen a performance difference earlier but couldn't prove it now.